recipes/reframed/meta.yaml

```diff
@@ -43,7 +45,6 @@ about:
 
 extra:
   recipe-maintainers:
+    - lhtxa
+    - cdanielmachado
     - acaprez
-  identifiers:
-    - biotools:reframed
-    - doi:10.5281/zenodo.7955995
```

Comment:
why do you remove those?

Reply:
Those are references where the package can be used but do not regard the package itself.

Reply:
They are back.